pyuvdata/uvdata/mir_meta_data.py

```diff
@@ -549,14 +559,14 @@ def __iter__(self):
 
     def __len__(self):
         """
-        Calculate the number entries in the data table.
+        Calculate the number of unmasked entries in the data table.
 
         Returns
         -------
         len : int
             Number of unique entries contained within the meta data.
         """
-        return self._data.size
+        return np.sum(self._mask)
```

Comment:
How many times does this get called when creating the `UVData` object in a normal `UVData.read` call? Wondering if it's worth having this cached.

Reply:
👍 Makes sense!